src/System.Windows.Forms/tests/InteropTests/PropertyGridTests.cs

```diff
@@ -27,7 +27,7 @@ public void ISupportErrorInfo_Supported_ButNoIErrorInfoGiven()
                 try
                 {
                     encodingEntry.SetPropertyTextValue("333");
-                    Assert.False(true, "Invalid set values should produce ExternalException which will be presenttted to the user.");
+                    Assert.False(true, "Invalid set values should produce ExternalException which will be presented to the user.");
```

Comment:
Took me a while to 'decipher' this Assert.
```suggestion
                    // We should never arrive here. If we do:
                    Assert.False(true, "Passing an invalid set of values should produce 'ExternalException' which will be presented to the user.");
```

Reply:
Yeah, I don't know why I didn't do this as Assert.Fail

Reply:
I'll turn this into an Assert.Fail in another PR.